apps/blast/install_blast.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+WORKING_DIR=/mnt/resource
+INSTALL_DIR=/mnt/resource
+
+# prerequistics and system update
+cd
+yum -y update
+wget https://aka.ms/downloadazcopy-v10-linux
```

Comment:
Please remove AZCOPY installaiton from the blast script as this is a dependency. There is alreay an install_azcopy.sh script that can be run in the installation

Reply:
Removed.